Failed tests (11634): TestStoreRangeMergeLastRange

The following test appears to have failed:

[#11634](https://circleci.com/gh/cockroachdb/cockroach/11634):

```
        01:19:24.022842  5.78µs      ·meta descriptor lookup                              kv/dist_sender.go:560
        01:19:24.022857  1.611657ms  ·sending RPC                                         kv/dist_sender.go:420
        01:19:24.024546  0           ·coordinator stops                                   kv/txn_coord_sender.go:495

I0120 01:19:24.025269 659 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestStoreRangeMergeLastRange (0.02s)
	client_merge_test.go:304: expected 'cannot merge final range' error; got <nil>
=== RUN   TestStoreRangeMergeNonCollocated
E0120 01:19:24.026797 659 gossip/gossip.go:210  different node IDs were set for the same gossip instance (2147483647, 1)
I0120 01:19:24.026968 659 raft/raft.go:446  [group 1] 1 became follower at term 5
I0120 01:19:24.027051 659 raft/raft.go:234  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0120 01:19:24.027095 659 raft/raft.go:526  [group 1] 1 is starting a new election at term 5
I0120 01:19:24.027137 659 raft/raft.go:459  [group 1] 1 became candidate at term 6
I0120 01:19:24.027176 659 raft/raft.go:508  [group 1] 1 received vote from 1 at term 6
I0120 01:19:24.027232 659 raft/raft.go:487  [group 1] 1 became leader at term 6
I0120 01:19:24.027534 659 storage/replica.go:1624  gossiping cluster id "" from store 1, range 1
--
I0120 01:19:38.644013 659 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestLogSplits (0.23s)
	leaktest.go:110: prior leak detected, leaktest disabled
=== RUN   Example_rebalancing
--- PASS: Example_rebalancing (0.12s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	16.578s
=== RUN   TestBatchBasics
I0120 01:19:20.451938 646 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.00s)
=== RUN   TestBatchGet
I0120 01:19:20.453215 646 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.00s)
=== RUN   TestBatchMerge
I0120 01:19:20.456123 646 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_merge_test.go
+++ b/storage/client_merge_test.go
@@ -295,6 +295,7 @@ func TestStoreRangeMergeWithData(t *testing.T) {
 // fails.
 func TestStoreRangeMergeLastRange(t *testing.T) {
 	defer leaktest.AfterTest(t)
+	defer config.TestingDisableTableSplits()()
 	store, stopper := createTestStore(t)
 	defer stopper.Stop()
 

